.github/workflows/release.yml

```diff
@@ -25,11 +31,11 @@ jobs:
       - name: Assemble eqwalizer binary
         run: "cd eqwalizer && native-image -H:IncludeResources=application.conf --no-server --no-fallback -jar target/scala-2.13/eqwalizer.jar eqwalizer"
```

Comment:
Here the native eqWAlizer image is not built with musl support, so it will still be linked dynamically against libc (see https://www.graalvm.org/22.0/reference-manual/native-image/StaticImages/).
Won't it be problematic?

Reply:
Seems fine indeed, I didn't realise that eqWAlizer was linking against glibc >2.17 only and that it was RHEL compatible.

We may come back to this and work on a fully static native image if the need arises.

Thanks for the check!